docs/redhat.rst

```diff
@@ -0,0 +1,59 @@
+
+.. _redhat_env:
+
+Red Hat Environment
+===================
+If you running Cekit in Red Hat internal infrastructure it behaves differently. This behavior is triggered by changing :ref:`redhat configuration option<redhat_config>` in Cekit configuration file.
+
+
+Tools
+-----
+Cekit integration with following tools is changed in following ways:
+
+* runs ``rhpkg`` instead of ``fedpkg``
+* runs ``odcs`` command with ``--redhat`` option set
+
+
+Environment Variables
+---------------------
+
+Following variables are added into the image:
+
+* ``JBOSS_IMAGE_NAME`` - contains name of the image
+* ``JBOSS_IMAGE_VERSION`` - contains version of the image
+
+Labels
+------
+
+Following labels are added into the image:
+
+* ``name`` - contains name of the image
+* ``version`` - contains version of the image
+* ``architecture`` - contains architecture of the image
+
+.. _redhat_repo:
+
+Repositories
+------------
+
+As we are building on not properly subscribed RHEL system and using internal repositories we need
```

Comment:
Rewrite this please. You should say only that we have custom way of accessing repositories that uses ODCS. Do not mention subscriptions.